mesonbuild/dependencies/misc.py

```diff
@@ -373,12 +373,10 @@ def __init__(self, environment, kwargs):
             self.version = self.VERSIONS[openmp_date]
             if self.clib_compiler.has_header('omp.h', '', self.env, dependencies=[self]):
                 self.is_found = True
+                self.compile_args = self.clib_compiler.openmp_flags()
```

Comment:
Does this go into the linker args as well?

Reply:
I tried that initially, but it resulted in `-fopenmp -fopenmp` in all link arguments. I guess it doesn't really hurt anything, but I know that nirbheek has been trying to delete duplicate arguments.

I guess I need to sort out the msvc problems first though.

Reply:
Specifically we get two `-fopenmps` because we use gcc as the linker, so it gets the -fopenmp for the compiler and the one for the linker.

Reply:
The answer is apparently "yes"